api/jobset/v1alpha2/jobset_webhook.go

```diff
@@ -178,8 +178,7 @@ func (js *JobSet) ValidateUpdate(old runtime.Object) (admission.Warnings, error)
 		}
 	}
 	// Note that SucccessPolicy and failurePolicy are made immutable via CEL.
-	errs := apivalidation.ValidateImmutableField(mungedSpec.ReplicatedJobs, oldJS.Spec.ReplicatedJobs, field.NewPath("spec").Child("replicatedJobs"))
```

Comment:
@ahg-g this is probably not what we want.

If we drop all validation of a replicated job, updates could add/remove entire replicated jobs.

I think we wouldn't want that.